src/llamacpp-chat/bentofile.yaml

```diff
@@ -1,14 +1,10 @@
 service: "service:LlamaCppChat"
 labels:
   source: https://github.com/bentoml/openllm-models-feed/tree/main/src/llamacpp-chat
```

Comment:
can you update this to `bentoml/openllm-models`?

Reply:
Sg!